importers/nmdb-import-cisco/Parser.cpp

```diff
@@ -417,6 +494,139 @@ Parser::addVlan(nmco::Vlan& vlan)
   d.vlans.push_back(vlan);
 }
 
+
+// Policy Related
+void
+Parser::createAccessGroup(nmco::InterfaceNetwork* iface,
+                          const std::string& bookName,
+                          const std::string& direction)
+{
+  usedRuleBooks[bookName] = {iface->getName(), direction};
+}
+
+void
+Parser::updateCurrentRuleBook(const std::string& name)
+{
+  curRuleBook = name;
+  curRuleId = 0;
+}
+
+void
+Parser::updateCurrentRule()
+{
+  ++curRuleId;
+  d.ruleBooks[curRuleBook][curRuleId].setRuleDescription(curRuleBook);
+}
+
+void
+Parser::setCurrentRuleAction(const std::string& action)
+{
+  d.ruleBooks[curRuleBook][curRuleId].setRuleId(curRuleId);
+  d.ruleBooks[curRuleBook][curRuleId].addAction(action);
+}
+
+void
+Parser::setCurrentRuleProtocol(const std::string& protocol)
+{
+  curRuleProtocol = protocol;
+}
+
+void
+Parser::setCurrentRuleSourcePorts(const std::string& ports)
+{
+  curRuleSourcePort = ports;
+}
+
+void
+Parser::setCurrentRuleDestinationPorts(const std::string& ports)
+{
+  curRuleDestinationPort = ports;
+}
+
+void
+Parser::setCurrentRuleSourceIpMask(nmco::IpAddress ipAddr,
+                                   const nmco::IpAddress& starMask)
+{
+  ipAddr.setWildcardNetmask(starMask);
+  const std::string ipAddrString {ipAddr.toString() + " *" + starMask.toString()};
```

Comment:
Make a //TODO note that this string will need to change when the setWildcardNetmask is properly implemented.